### PR TITLE
testing/openresty: upgrade to 1.15.8.2

### PR DESCRIPTION
**Pull request by tcely**

### CODE REVIEW

**Comment by Ikke** [comment]
@tcely Can you clean these commits up a bit? There are several upgrades and one upgrade revert in this single PR.

**Comment by tcely** [comment]
@Ikke I squashed the RC upgrades into a single commit.

**Comment by Ikke** [comment]
And why is that revert there? (Would be nice to have it included in the commit message)

**Comment by tcely** [comment]
The revert was there to preserve clean history and diffs, but since the commits are squashed now those are being list anyway.

File: main/nginx/APKBUILD [hunk 1]
```diff
@@ -304,7 +306,7 @@ _module() {
 	sonames="${sonames:-"ngx_${name}_module.so"}"
 
 	pkgdesc="$pkgdesc (module $name)"
-	depends="$pkgname $(eval "echo \$_${name}_depends")"
+	depends="$_provide $(eval "echo \$_${name}_depends")"
```
Comment (ncopa) [inline main/nginx/APKBUILD]: @jirutka what do you think about this? both `openresty` and `nginx` will provide `cmd:nginx` but there is no guarantee which implementation the module will pull in.

File: testing/openresty/APKBUILD [hunk 2]
```diff
@@ -139,7 +140,7 @@ _module() {
 	local _deps="$(eval echo "\$_module_${name//-/_}_deps")"
 
 	pkgdesc="$pkgdesc (module $soname)"
-	depends="cmd:nginx${_deps:+ }${_deps}"
+	depends="${_provide}${_deps:+ }${_deps}"
```
Comment (ncopa) [inline testing/openresty/APKBUILD]: will `openresty` (1.15) modules work with `nginx` (1.16) and vice versa?

Wouldn't it be better to depend on the exact nginx implementation?

File: testing/openresty/APKBUILD [hunk 3]
```diff
@@ -1,9 +1,10 @@
 # Contributor: tcely <openresty+aports@tcely.33mail.com>
 # Maintainer: tcely <openresty+aports@tcely.33mail.com>
-pkgname=openresty
-pkgver="1.15.8.1"
+_upstream="openresty"
```
Comment (ncopa) [inline testing/openresty/APKBUILD]: IMHO, the `_upstream` variable here is meaningless and should be avoided.

File: testing/openresty/GPGKEYS [hunk 4]
```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd" >
```
Comment (ncopa) [inline testing/openresty/GPGKEYS]: Do we really need to store this in html format?

File: testing/openresty/APKBUILD [hunk 5]
```diff
@@ -32,17 +34,33 @@ _add_module() {
...
+_add_module xslt-filter
 
 prepare() {
 	default_prepare
 	cd "$builddir"
 
 	sed -i.orig -e '/--add-module=$dir/s/module/dynamic-&/' configure
+
+	local tag='LuaJIT-2.1'
+	cd "bundle/${tag}"-*
```
Comment (ncopa) [inline testing/openresty/APKBUILD]: Could we use system `luajit` instead of the bundled?

**Pull request closed without merging**